NEWS.md

```diff
@@ -1,3 +1,8 @@
+# redcaptools 0.3.2
+
+* rename `redcap_select_rename` to `redcap_import_select` and `redcap_recode` to `redcap_import_recode`
```

Comment:
perhaps rather something along the lines of "new function redcap_import_recode"?

Reply:
not sure if I understand correctly. i added your suggestion